src/behaviours/application-update/installing-update.test.ts

```diff
@@ -108,7 +103,7 @@ describe("installing update", () => {
         });
 
         it("notifies the user", () => {
-          expect(showInfoNotificationMock).toHaveBeenCalledWith("No new updates available");
+          expect(rendered.getByTestId("auto-update-component")).toHaveTextContent("No new updates available");
```

Comment:
If we used different `test-id` for each of these, we wouldn't have to `toHaveTextContent`. We could just `toBeInTheDocument()` which would make this test less prone to fail for wrong reason (e.g. we are changing notification text)